framework/console/controllers/QueueController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\console\controllers;
+
+use Yii;
+use yii\console\Controller;
+
+/**
+ * Runs the queue listener.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+class QueueController extends Controller
```

Comment:
> Usually queues are processed by deamons - not console commands.

Ok, so you think daemon on php is impossible. Ok, maybe its time to start checking other queue implementations on php before provice such strange feedback?